app/panels.php

```diff
@@ -1,144 +1,105 @@
 <?php namespace AgreablePromoPlugin;
 
 use AgreablePromoPlugin\Helper;
-// $ns = Helper::get('agreable_namespace');
-$ns = 'agreable_promo';
+\add_action('plugins_loaded', function() {
+  $ns = 'agreable_promo';
+  $user = wp_get_current_user();
+  $user_roles = $user->roles;
+  if (in_array('promos_editor', $user_roles) || in_array('administrator',$user_roles)) {
```

Comment:
awesome, i was trying to find something like this but was probs searching the wrong thing. thanking you